src/cascadia/TerminalApp/Resources/en-US/ContextMenu.resw

```diff
@@ -117,6 +117,33 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AppName" xml:space="preserve">
+    <value>Terminal</value>
+  </data>
+  <data name="AppNameDev" xml:space="preserve">
+    <value>Terminal Dev</value>
+  </data>
+  <data name="AppNamePre" xml:space="preserve">
+    <value>Terminal Preview</value>
+  </data>
+  <data name="AppShortName" xml:space="preserve">
+    <value>Terminal</value>
+  </data>
+  <data name="AppShortNameDev" xml:space="preserve">
+    <value>Terminal Dev</value>
+  </data>
+  <data name="AppShortNamePre" xml:space="preserve">
+    <value>Terminal Preview</value>
+  </data>
+  <data name="AppDescription" xml:space="preserve">
+    <value>The New Windows Terminal</value>

```

Comment:
Is it really "new" anymore?

<img width="240" alt="image" src="https://user-images.githubusercontent.com/18221333/151249429-149b89a8-05e5-4d31-ba76-c1424945df38.png">

Reply:
> Is it really "new" anymore?

Maybe not, it's just to distinguish it from the "old" Terminal https://guidebookgallery.org/screenshots/terminal#win101

(j/k)

Reply:
New goal for #3327: Make the Terminal be able to look like that monstrosity 😆